CONTRIBUTING.md

```diff
@@ -1,19 +1,41 @@
+[![ColPrac: Contributor's Guide on Collaborative Practices for Community Packages](https://img.shields.io/badge/ColPrac-Contributor's%20Guide-blueviolet)](https://github.com/SciML/ColPrac)
+
+We follow the ColPrac guide for collaborative practices. New contributor should make sure to read that guide.
+Following here are some additional clarifications and practices we follow.
+
 Thanks for taking the plunge!
 
 ## Reporting Issues
 
-* It's always good to start with a quick search for an existing issue to post on, or related issues for context, before opening a new issue
-* Including minimal examples is greatly appeciated
-* If it's a bug, or unexpected behaviour, reproducing on the latest development version (`Pkg.checkout("DataFrames")`) is a good gut check and can streamline the process, along with including the first two lines of output from `versioninfo()`
+* It's always good to start with a quick search for an existing issue to post on,
+  or related issues for context, before opening a new issue
+* Including minimal examples is greatly appreciated
+* If it's a bug, or unexpected behaviour, reproducing on the latest development version
+  (`Pkg.checkout("DataFrames")`) is a good gut check and can streamline the process,
```

Comment:
This doesn't exist anymore

Reply:
Thank you!

Reply:
`Pkg.add(name="DataFrames", branch="master")` maybe is a good replacement.

Reply:
ah - in parallel I have opened a PR with `Pkg.develop` recommendation. Can you please answer there what you prefer. Thank you!